packages/react-native-editor/.watchmanconfig

```diff
@@ -1 +1,3 @@
-{}
+{
+	"content_hash_max_items": 400000
```

Comment:
This number was obtained by running `find . | wc -l` from the gutenberg-mobile root